gluecodium/src/test/resources/smoke/listeners/output/swift/smoke/ListenerInterface.swift

```diff
@@ -0,0 +1,132 @@
+//
+//
+import Foundation
+public protocol ListenerInterface : AnyObject {
+    func notify() -> Void
+}
+internal class _ListenerInterface: ListenerInterface {
+    let c_instance : _baseRef
+    init(cListenerInterface: _baseRef) {
+        guard cListenerInterface != 0 else {
+            fatalError("Nullptr value is not supported for initializers")
+        }
+        c_instance = cListenerInterface
+    }
+    deinit {
+        smoke_ListenerInterface_remove_swift_object_from_wrapper_cache(c_instance)
+        smoke_ListenerInterface_release_handle(c_instance)
+    }
+    public func notify() -> Void {
+        return moveFromCType(smoke_ListenerInterface_notify(self.c_instance))
+    }
+}
+@_cdecl("_CBridgeInitsmoke_ListenerInterface")
+internal func _CBridgeInitsmoke_ListenerInterface(handle: _baseRef) -> UnsafeMutableRawPointer {
+    let reference = _ListenerInterface(cListenerInterface: handle)
+    return Unmanaged<AnyObject>.passRetained(reference).toOpaque()
+}
+internal class ListenerInterface_WeakHolder {
+    weak var value: ListenerInterface?
+    init(_ value: ListenerInterface) {
+        self.value = value
+    }
+}
+internal func getRef(_ ref: ListenerInterface?, owning: Bool = true, weak: Bool = false) -> RefHolder {
+    guard let reference = ref else {
+        return RefHolder(0)
+    }
+    if let instanceReference = reference as? NativeBase {
+        let handle_copy = smoke_ListenerInterface_copy_handle(instanceReference.c_handle)
+        return owning
+            ? RefHolder(ref: handle_copy, release: smoke_ListenerInterface_release_handle)
+            : RefHolder(handle_copy)
+    }
+    var functions = smoke_ListenerInterface_FunctionTable()
+    let strongRef: AnyObject = weak ? ListenerInterface_WeakHolder(reference) : reference
+    functions.swift_pointer = Unmanaged<AnyObject>.passRetained(strongRef).toOpaque()
+    functions.release = {swift_class_pointer in
+        if let swift_class = swift_class_pointer {
+            Unmanaged<AnyObject>.fromOpaque(swift_class).release()
+        }
+    }
+    functions.smoke_ListenerInterface_notify = {(swift_class_pointer) in
+        let unretained_value = Unmanaged<AnyObject>.fromOpaque(swift_class_pointer!).takeUnretainedValue()
+        let swift_class: ListenerInterface
+        if (unretained_value is ListenerInterface_WeakHolder) {
+            if let weak_value = (unretained_value as! ListenerInterface_WeakHolder).value {
+                swift_class = weak_value
+            } else {
+                return
```

Comment:
This call from C++ to Swift, right? I'm not sure that it should be just silently skipped

Reply:
The consensus is to introduce additional validation that limits the methods of such interface to void-returning only.